app/views/users/_admin_change_username.html.erb

```diff
@@ -0,0 +1,27 @@
+<h2 class="heading"><%= t(".page_heading") %></h2>
+<%= error_messages_for :user %>
+<p class="note"><%= t(".description") %></p>
+
+<ul class="navigation actions" role="navigation">
```

Comment:
Can you get rid of the `role="navigation"` attribute and value here and in the user version of this page? It's invalid on `ul` items and should be removed whenever you see it.